pkg/distributor/tenant_topic_tee.go

```diff
@@ -304,6 +305,8 @@ func NewTenantTopicWriter(
 		return nil, fmt.Errorf("unknown strategy %q", strategy.String())
 	}
 
+	reg = prometheus.WrapRegistererWithPrefix("tenant_topic_tee_", reg)
```

Comment:
this would also add the prefix to the producer metrics, is this intentional? an alternative could be to register with a constant label that says `producer="tenant_topic_tee"`